Create cross-browser-test.md (#68)
atolye15/checklist: checklists/crossbrowser-test.md

---

> title: Crossbrowser Test

Reviewer: ```suggestion
title: Cross-Browser Test
```

category: Project Management
date: "2020-03-11"
tags: ['test', 'responsive']

> description: Complete the crossbrowser test to check your project works correctly in different types of devices and browsers.

Reviewer: ```suggestion
description: Complete the cross-browser test to check if your project works correctly in different types of devices and browsers.
```

---

> - [ ] Test the project via different browsers on desktop  

Reviewer: ```suggestion
- [ ] Test the project with different browsers on desktop  
```

> We do crossbrowser test on desktop side via different browsers such as Safari, Chrome, Firefox.  

Reviewer: ```suggestion
We do cross-browser test on desktop with different browsers such as Safari, Chrome, Firefox.  
```

- [ ] Test the project via a bigger screen monitor  

> We control the design of project is suitable for many measure or not.  

Reviewer 2: ```suggestion
- [ ] Test the project in bigger screen size    
So that you can start to check the responsiveness of the website. We should also control the design of the project in different screen sizes.
```

> - [ ] Test the project via different browsers on IOS  

Reviewer: ```suggestion
- [ ] Test the project with different browsers on iOS  
```

> We do crossbrowser test on IOS mobile device via different browsers such as Safari, Chrome, Firefox.  

Reviewer 2: ```suggestion
- [ ] Test the project via different browsers on iOS device  
Especially check the project works correctly on Safari, and other browsers.  
```

- [ ] Test the project via different browsers on Android  

> We do crossbrowser test on Android mobile device via different browsers such as Chrome, Firefox.  

Reviewer: ```suggestion
- [ ] Test the project with different browsers on Android  
We do the cross-browser tests on Android mobile devices via different browsers such as Chrome, Firefox.  
```

- [ ] Test the project via different browser on Tablet  

> We do crossbrowser test on Tablet device via different browsers such as Safari, Chrome, Firefox.

Reviewer 2: ```suggestion
- [ ] Test the project in different browser on Tablet such as Safari, Chrome  
```
